accessclinicaldata.niaid.nih.gov/portal/gitops.json

```diff
@@ -15,19 +15,22 @@
-          "<b>How to View Available Data</b>",
-          "Go to <a href='https://accessclinicaldata.niaid.nih.gov/study-viewer/clinical_trials'>View Clinical Trials</a>. For each study listed, open the details and select the \"Learn More\" button to review a description of the clinical trial and data dictionary.",
-          "<b>How to Request Data</b>",
+          "<h3>How to View Available Data</h3>",
+          "<p>Click the button below and for each study listed, open the details and select the \"Learn More\" button to review a description of the clinical trial and data dictionary.</p><div class='nct-btn mr-5'><a class='flex flex-row items-center' href='https://qa-niaid.planx-pla.net/study-viewer/clinical_trials'>View Clinical Trials</a></div></div><br />",
+          "<h3>How to Request Data</h3>",
           "Researchers looking to access NIAID COVID-19 and other sponsored clinical trials data may start the request process by completing the following steps:",
           "1. On the study page of the data set you are interested in, select the \"Login to Request Access\" button to login to the AccessClinicalData@NIAID platform and fill in and submit the Data Access Request (DAR) online form.",
           "2. Your DAR will be reviewed by the NIAID Clinical Trials Data Access Committee. ",
           "3. Upon approval of the DAR, the primary data requestor and their institution will be notified and required to agree to and sign a NIAID Data Use Agreement (DUA) online via DocuSign that outlines the terms of the use of the data.",
           "4. Upon NIAID’s receipt of the NIAID DUA, data requestors will be notified on how to download the approved data.",
-          "5. Please note you can review a blank <a href='https://accessclinicaldata.niaid.nih.gov/dashboard/Public/files/NIAID_DAR.pdf'>DAR</a> and <a href='https://accessclinicaldata.niaid.nih.gov/dashboard/Public/files/NIAIDDUA2023Accessclinicaldata@NIAID.pdf'>DUA</a> prior to viewing and requesting the available data, but the request must be done using a DAR online form found on the data platform, as outlined above.",
-          "<b>About the Platform</b>",
+          "5. Please note you can review a blank <a href='https://qa-niaid.planx-pla.net/dashboard/Public/files/NIAID_DAR.pdf'>DAR</a> and <a href='https://qa-niaid.planx-pla.net/dashboard/Public/files/NIAIDDUA2021Accessclinicaldata@NIAID.pdf'>DUA</a> prior to viewing and requesting the available data, but the request must be done using a DAR online form found on the data platform, as outlined above.",
```

Comment:
need to change this back

Reply:
i think we could use relative urls instead of absolute, but that's a problem for another day

Reply:
Thanks Pauline.

Reply:
Good catch, thanks! Didn't notice those hardcoded urls there. Fixed in latest push.